packages/hero/index.js

```diff
@@ -32,14 +32,15 @@ function Hero({ data, centered, gaPrefix, className, videoControlsTop }) {
   const hasVideos = videos && videos.length > 0
   const { themeClass } = useProductMeta(product)
   const h1OrphanCount = 9
+  const isCentered = centered || data.centered
 
   return (
     <div
       className={classNames(
         s.root,
         s[backgroundTheme],
         themeClass,
-        { centered: centered || data.centered },
+        { [s.centered]: isCentered },
```

Comment:
This `hero` `centered` stuff should have been implemented in #299, but I missed it 😭